README.rst

```diff
@@ -27,6 +27,31 @@ Running flyteconsole locally requires `NodeJS <https://nodejs.org>`_ and
 `yarn <https://yarnpkg.com>`_. Once these are installed, all of the dependencies
 can be installed by running ``yarn`` in the project directory.
 
+=====================
+Quickstart
+====================
+
+#. Follow `Start a Local flyte backend <https://docs.flyte.org/en/latest/getting_started/first_run.html>`_, like
+
+  .. code:: bash
+
+     docker run --rm --privileged -p 30081:30081 -p 30082:30082 -p 30084:30084 ghcr.io/flyteorg/flyte-sandbox
+
+#. Now Export 2 env variables
+
+   .. code:: bash
+
+       export ADMIN_API_URL=http://localhost:30081
```

Comment:
I recommend putting these values into a `.env` file so they persist across shells. But this is sufficient to get started.

Reply:
ok thats a good advice, let me update the docs